src/freeze-dry/inline-styles.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env jest */
+
+import inlineStyles from 'src/freeze-dry/inline-styles'
+import * as common from 'src/freeze-dry/common'
+
+fetch.mockResponse(new Blob(['Dummy Styles']), { 'status': 200, 'statusText': 'Response Accepted' })
+
+describe('inlineStyles', () => {
+    test('should return create a <style> with the fetch of stylesheet', async () => {
+        const parser = new DOMParser()
+        const rootElement = parser.parseFromString(
+            '<html><head><link rel="stylesheet" type="text/css" href="https://example.com/theme.css"></head></html>',
+            'text/html'
+        )
+        const docUrl = 'https://example.com'
+        await inlineStyles({rootElement, docUrl})
+        expect(rootElement.querySelector('style').innerHTML).toBe('Dummy Styles')
+    })
```

Comment:
Great, this is the kind of test I mean with passing an example doc and checking the result.